nx-X11/lib/X11/Imakefile

```diff
@@ -1063,13 +1060,12 @@ EXTRASHAREDOBJS = $(STATICLCOBJS)
 
 SpecialCLibObjectRule(ErrDes,$(ICONFIGFILES),$(EDB_DEFINES))
 SpecialCLibObjectRule(StrKeysym,$(ICONFIGFILES),$(KDB_DEFINES))
-SpecialCLibObjectRule(ConnDis,$(ICONFIGFILES),$(XDMAUTHDEFS) $(RPCDEFS) $(CONN_DEFINES) $(SOCK_DEFINES) $(POLL_DEFINES) $(XTRANS_X_DEFINES) $(K5INCL) $(K5DEFS))
```

Comment:
`XDMAUTHDEFS` is still defined as `-DHASXDMAUTH`. Should it really be removed here, or the whole `#if HasXdmAuth` section above be removed, too?